src/aave-v2/MorphoGovernance.sol

```diff
@@ -360,8 +363,10 @@ abstract contract MorphoGovernance is MorphoUtils {
         onlyOwner
         isMarketCreated(_poolToken)
     {
-        if (!marketPauseStatus[_poolToken].isBorrowPaused) revert BorrowNotPaused();
-        marketPauseStatus[_poolToken].isDeprecated = _isDeprecated;
+        Types.MarketPauseStatus storage status = marketPauseStatus[_poolToken];
+        if (!status.isBorrowPaused) revert BorrowNotPaused();
+        if (_isDeprecated && status.isLiquidateBorrowPaused) revert LiquidateBorrowIsPaused();
```

Comment:
I wrote a comment in the issue about this check: https://github.com/spearbit-audits/review-morpho-november/issues/73#issuecomment-1434722223